scaleway/resource_k8s_cluster_beta.go

```diff
@@ -210,6 +216,39 @@ func resourceScalewayK8SClusterBeta() *schema.Resource {
 							Computed:    true,
 							Description: "The date and time of the last update of the default pool",
 						},
+						"nodes": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"name": {
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "The name of the node",
+									},
+									"pool_id": {
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "The pool ID whose the node belongs to",
```

Comment:
`The pool ID whose the node belongs to` -> `The ID of the pool the node belongs to`

Reply:
Will change